src/main/java/games/strategy/engine/lobby/server/login/LobbyLoginValidator.java

```diff
@@ -121,33 +126,31 @@ public LobbyLoginValidator() {
     return challenge;
   }
 
+  @Nullable
   @Override
-  public String verifyConnection(final Map<String, String> propertiesSentToClient,
-      final Map<String, String> propertiesReadFromClient, final String clientName, final String clientMac,
+  public String verifyConnection(
+      final Map<String, String> challenge,
+      final Map<String, String> response,
+      final String clientName,
+      final String clientMac,
       final SocketAddress remoteAddress) {
     if (lobbyPropertyReader.isMaintenanceMode()) {
       return ErrorMessages.MAINTENANCE_MODE_ENABLED;
     }
 
-    final String error = verifyConnectionInternal(propertiesReadFromClient, clientName, clientMac);
-    if (error != null) {
-      logger.info("Bad login attempt from " + remoteAddress + " for user " + clientName + " error:" + error);
```

Comment:
I removed the `Logger` calls in `LobbyLoginValidator`, as they basically repeat the same information logged by `CompositeAccessLog`.

Reply:
Thanks for working on this!